### DISCUSSION TITLE
Hold note tails give score regardless of cursor/thumb position

### DISCUSSION THREAD

**Discussion by ziv-vy** [LumpBloom7/sentakki#630] [Ideas]
https://github.com/user-attachments/assets/962eab5a-ca53-40b3-adb4-4836c5143b45

I don't think this is a bug but maybe it probably should be removed.

**Comment by LumpBloom7** [comment]
This is by design. (It's intentionally trying to mimic the behavior of the touch sensor of mania) 

"Button" lane input method in sentakki settings is not the ideal way to play, and most gameplay testing is done using sensor mode. Prolly should outright remove button mode at some point.

**Comment by ziv-vy** [comment]
Lump forgot to close it, so I'll close it myself.

**Reply by LumpBloom7** [reply]
Thank you
